### ISSUE TITLE
[COMPLETED] Helpful idea...

### ISSUE THREAD

**Issue by stingersplash** [thecoder75/liveme-pro-tools#101]
Just wanted to say thank you for adding the new bookmark feature, it works better then I thought it would this early in development.
So previously I recommended a helpful idea...
So when you click on a video to play, a green eyeball on the left of the video shows up only after you refresh the accounts page. Would it be possible to have the green eyeball popup immediately after you play a video instead of having to refresh the page? Lets say an account has 200 replays, and you loose track of which replay you just watched, you would need to refresh all 200 replays to find where you left off.
And you mentioned...
I could possibly add an event receiver that would allow setting/changing the eye icon when a replay is viewed.
My question is, is it still possible to implement this? or does it not work correctly?
Thanks again

**Comment by thecoder75** [comment]
This is currently already done in the code, I just haven't built a release since adding this and a few other improvements.
